src/JuliaInterpreter.jl

```diff
@@ -61,6 +61,7 @@ struct JuliaFrameCode
     scope::Union{Method,Module}
     code::CodeInfo
     methodtables::Vector{Union{Compiled,TypeMapEntry}} # line-by-line method tables for generic-function :call Exprs
+    breakpoints::Vector{Tuple{Bool,Function}}          # (isactive, condition)
```

Comment:
Could potentially make this a named tuple `(isactive = true, f = ...)` for a bit nicer usage later `bp.isactive` over `bp[1]`.